blocks/pocketguide/pocketguide.css

```diff
@@ -0,0 +1,44 @@
+.pocketguide {
+    text-align:center;
+    position:relative;
+    padding-top:136px;
+    border-radius:10px;
+    border:1px solid #7f5732;
+    margin:100px 15px;
+    min-width: 200px;
+}
+
+.pocketguide>*>*>h2 {
+    text-transform: none;
+}
+
+.pocketguide>div>div>p:nth-child(4)>picture{
+    height:190.8px;
+    width:127.4px;
+    right:calc(50% - 63.7px);
+    top:-60px;
+    position:absolute;
+    transform:rotate(13deg)
+}
+
+@media only screen and (min-width:1000px) {
```

Comment:
I think you are missing another breakpoint at 1200px.

Reply:
dont think so, there's breakpoints for 1000 related to pocketguide only. 
however, there's BPs for 1138, 1200 and 1800 for the home, hero and desktop banner. not related to pocketguide though.

Reply:
The text-align left seems to happen after 1200px - not at 1000px already, no?